recipes/libuv/all/conanfile.py

```diff
@@ -72,7 +72,7 @@ def package(self):
         cmake.install()
 
     def package_info(self):
-        self.cpp_info.names["pkg_config"] = "libuv"
+        self.cpp_info.set_property("pkg_config_name", "libuv")
```

Comment:
```suggestion
        self.cpp_info.set_property("pkg_config_name", "libuv")
        self.cpp_info.set_property("cmake_file_name", "LibUV")
        self.cpp_info.set_property("cmake_target_name", "LibUV::LibUV")
        # TODO: Remove after Conan 2.0
        self.cpp_info.name["cmake_find_package"] = "LibUV"
        self.cpp_info.name["cmake_find_package_multi"] = "LibUV"
```
Cmake official modules have libuv listed as LibUV: https://gitlab.kitware.com/cmake/cmake/-/blob/v3.22.1/Source/Modules/FindLibUV.cmake

Reply:
If it's not listed in https://cmake.org/cmake/help/latest/manual/cmake-modules.7.html, it's not official. The FindLibUV you are showing is a internal build detail of CMake.